bin/sdl_make_tracklooper

```diff
@@ -158,10 +167,15 @@ else
     (cd SDL && make clean && make ${T3T3EXTENSIONOPT} ${T5CUTOPT} ${BACKENDOPT} ${NOPLSDUPCLEANOPT} -j 32 ${MAKETARGET} && cd -) >> ${LOG} 2>&1
 fi
 
-if [ ! -f SDL/libsdl.so ]; then
-    echo "ERROR: SDL/libsdl.so failed to compile!" | tee -a ${LOG}
-    echo "See ${LOG} file for more detail..." | tee -a ${LOG}
-    exit 1
+if [[ "$BACKENDOPT" != *"gpu"* ]] && [ ! -f SDL/libsdl_cpu.so ]; then
```

Comment:
Doesn't make more sense to make these equalities rather than inequalities? I imagine that adding a third option would not be possible as the logic is now.

Reply:
Yeah, I'll do that.